mongo/options/listcollectionsoptions.go

```diff
@@ -32,6 +36,12 @@ func (lc *ListCollectionsOptions) SetBatchSize(size int32) *ListCollectionsOptio
 	return lc
 }
 
+// SetAuthorizedCollections sets the value for the AuthorizedCollections field.
```

Comment:
```suggestion
// SetAuthorizedCollections sets the value for the AuthorizedCollections field.
// This option is only valid for MongoDB server versions >= 4.0.
// Server versions < 4.0 ignore this option.
```

The spec does not request documenting this. The Go driver can still connect to pre 4.0 servers until the minimum wire version is bumped to 6. I suggest noting to save users some headache if they are attempting to use this option on pre 4.0 servers.